File: Modul-3/Python/PRAK301_2410817210001_MuhammadFadillah.py
angka = input("Masukkan tiga angka dipisahkan dengan spasi: ").split()

a, b, c = map(int, angka)

if a > b:
    a, b = b, a
if a > c:
    a, c = c, a
if b > c:
    b, c = c, b

print(f"Hasil pengurutan: {a} {b} {c}")

File: Modul-3/C/PRAK301_2410817210001_MuhammadFadillah.c
#include <stdio.h>

int main() {
    int a, b, c;

    printf("Masukkan tiga angka dipisahkan dengan spasi: ");
    scanf("%d %d %d", &a, &b, &c);

    if (a > b) {
        int temp = a;
        a = b;
        b = temp;
    }
    if (a > c) {
        int temp = a;
        a = c;
        c = temp;
    }
    if (b > c) {
        int temp = b;
        b = c;
        c = temp;
    }

    printf("Hasil pengurutan: %d %d %d\n", a, b, c);

    return 0;
}
